packages/opentelemetry-cloud-trace-exporter/package.json

```diff
@@ -56,6 +56,7 @@
     "@opentelemetry/core": "^0.7.0",
     "@opentelemetry/resources": "^0.7.0",
     "@opentelemetry/tracing": "^0.7.0",
+    "gcp-metadata": "^4.1.0",
```

Comment:
My understanding is that the `gcp-metadata` package is only used by the tests, in which case it should be part of "devDependencies".

Reply:
+1